src/test/java/integration/BaseTest.java

```diff
@@ -22,78 +22,86 @@ enum AuthorizationType {
 }
 
 public class BaseTest {
-    protected static String spKey;
+    protected static String servicePrincipalKey;
     protected static AccessKey accessKey;
-    protected static String repoId;
+    protected static String repositoryId;
     protected static Map<String, String> testHeaders;
     protected static RepositoryApiClient repositoryApiClient;
+    protected static String testHeaderValue;
     protected static String username;
     protected static String password;
     protected static String baseUrl;
```

Comment:
Does it make sense to make the rest of field `private static final`?

Reply:
yea so i think it probably makes more sense to make the rest of the fields `private static` other than the `repositoryId` and `repositoryApiClient` and since the remaining variables cannot be initialized right away and need to be assigned at certain parts of the implementation. Both the `repositoryId` and `repositoryApiClient` variables are used by the subclasses which is why i prefer to keep it at protected

Reply:
Got it. Thanks for the explanation.